.changes/unreleased/Fixes-20241030-130239.yaml

```
@@ -0,0 +1,6 @@
+kind: Fixes
```

We're considering this a [feature rather than a fix](https://github.com/dbt-labs/dbt-core/issues/10948#issuecomment-2452219657), so could you make this update?

```suggestion
kind: Features
```